lib/experimental/Navigation/Omnibutton/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { Icon } from "@/components/Utilities/Icon"
+import { Dropdown, DropdownItem } from "@/experimental/Navigation/Dropdown"
+import { Question } from "@/icons/app"
+import { cn, focusRing } from "@/lib/utils"
+
+interface Option {
+  title: string
+  description?: string
+  href?: string
+  onClick?: () => void
+  icon?: string
+  target?: string
+}
+
+interface OmniButtonProps {
+  label: string
+  options: Option[]
```

Comment:
do we always have options or there is also one action scenario?

Reply:
As far as I understand, yes, [there is always options](https://github.com/factorialco/factorial/blob/316becfcad90aaf7b10322811cc7c63f70a7bc49/frontend/src/modules/onboarding/components/OmniButton/getOptions.ts#L4).